install_config/install/docker_registry.adoc

```diff
@@ -20,26 +20,47 @@ link:../../architecture/infrastructure_components/image_registry.html#integrated
 Docker registry] that can be deployed in your OpenShift environment to locally
 manage images.
 
-To make full use of the registry, you must create a
-link:../../architecture/core_concepts/projects_and_users.html#users[regular user]
-with the *system:registry* role, if one does not already exist, using
-your preferred
-link:../../admin_guide/configuring_authentication.html[identity provider].
-This is because
-link:../../architecture/core_concepts/projects_and_users.html#users[system users],
-such as *system:admin*, cannot obtain access tokens, and therefore
+[[prerequisites-for-access]]
+=== Prerequisites for Access
+To access the registry directly, you must:
+
+- Create a
+link:../../architecture/core_concepts/projects_and_users.html#users[regular
+user] with the *system:registry* role, if one does not already exist, using your
+preferred link:../../admin_guide/configuring_authentication.html[identity
+provider]. A regular user can generate an access token required for logging in to
+the registry.
+link:../../architecture/core_concepts/projects_and_users.html#users[System
+users], such as *system:admin*, cannot obtain access tokens and, therefore,
 cannot access the registry directly.
-
++
 For example, if you are using `HTPASSWD` authentication:
-
++
 ----
 # htpasswd /etc/origin/openshift-htpasswd reguser
 # oadm policy add-role-to-user system:registry reguser
 ----
 
+- Have the *system:image-builder* role. To add this role:
++
+----
+$ oadm policy add-role-to-user system:image-builder <user>
+----
+
+- Have the *admin* role for the project associated with the Docker operation. For
+example, if accessing images in the global *openshift* project:
++
+----
+
+ $ oadm policy add-role-to-user admin <user> -n openshift
+----
+
+
 For more information, see
-link:../../install_config/configuring_authentication.html#HTPasswdPasswordIdentityProvider[Configuring Authentication]
-and link:../../admin_guide/manage_authorization_policy.html#managing-role-bindings[Managing Role Bindings].
+link:../../install_config/configuring_authentication.html#HTPasswdPasswordIdentityProvider[Configuring
+Authentication] and
+link:../../admin_guide/manage_authorization_policy.html#managing-role-bindings[Managing
+Role Bindings].
```

Comment:
We could possibly lose this sentence now; it was previously associated w/ just the first bullet, which already points to Config Auth in an inline link. Although, the Managing Role Bindings section is applicable to all 3 bullets, so maybe just drop the mention of Config Auth here and keep it simply "For more information, see Managing Role Bindings".